notebooks/byod.py

```diff
@@ -98,27 +98,33 @@ def upload_columns(table: str, columns: Dict[str, List[Any]]):
             tsv_data += os.linesep + "\t".join([f"{i}", *[columns[h][i] for h in column_headers]])
         upload_data_table(tsv_data)
 
+    def parse_cram_crai(filename: str):
+        sample, _ = filename.split(".", 1)                                                                    
+        if filename.endswith(".cram"): 
+            ext = "cram" 
+        elif filename.endswith(".crai") or filename.endswith(".cram.crai"):
+            ext = "crai" 
+        else:
+            # Metadata etc often takes the form of *.txt files so we'll ignore
+            # it rather than raising an exception
+            print(f"Unable to parse {filename} into sample and extension")
```

Comment:
I think it's appropriate to silently skip non-cram-crai files.

We can add documentation somewhere indicating that other file types will be ignored.

Reply:
Line 46 already say this will happen so I'll take out the print.